pajbot/apiwrappers/authentication/access_token.py

```diff
@@ -146,3 +146,11 @@ def can_refresh(self):
 
     def refresh(self, api):
         return api.get_app_access_token(self.scope)
+
+
+class SpotifyAccessToken(AccessToken):
+    def can_refresh(self):
+        return True
+
+    def refresh(self, api):
+        return api.refresh_user_access_token(self.refresh_token)
```

Comment:
This should not be in this file, see how twitch apiwrappers are contained in their own folder